### PR TITLE
style(sql): make migrations more greppable

### PR DESCRIPTION
**Pull request by mgaffney**

### CODE REVIEW

**Review by johanbrandhorst — APPROVED** [review]
Mostly 🙈 stamp, but I'd appreciate if we could write up a basic style guide for this for new contributions too, unless we already have one somewhere?

**Comment by mgaffney** [comment]
@johanbrandhorst I agree. I will add it to this PR.

**Review by tmessi — APPROVED** [review]

File: sql-style-guide.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
+# SQL Style Guide
+
+This guide establishes the standards for SQL in Boundary.
```
Comment (jimlambrt) [inline sql-style-guide.md:3]: perhaps `SQL migrations`

**Review by jimlambrt — COMMENTED** [review]
Just a suggestion on the doc.

File: sql-style-guide.md (hunk 1 above)
Comment (mgaffney) [reply sql-style-guide.md:3]: The goal is that it will eventually cover all SQL.

File: sql-style-guide.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
+# SQL Style Guide
```
Comment (tmessi) [inline sql-style-guide.md:1]: Do we have a style/naming guidelines for create/drop index?

Comment (mgaffney) [reply sql-style-guide.md:1]: No, not yet

**Review by louisruch — APPROVED** [review]
Thks!

**Review by ddebko — APPROVED** [review]
LGTM

**Review by johanbrandhorst — APPROVED** [review]

**Pull request merged**